src/index.js

```diff
@@ -1,8 +1,42 @@
+const si = require('systeminformation');
+
 /**
  * A module for the server-state system
- * @returns A JSON-serializable (via `JSON.stringify()`) version information about the server state
+ * 
+ * A module to view characteristics of your system
+ * 
+ * @throws if anything than 'object' given as options
+ * @throws if the key in options is not a function of the systeminformation module
+ * @throws if an invalid filter is given for the specified function
+ * 
+ * @argument options object with functions for the systeminformation module to check as keys and 
```

Comment:
This should be `@param`... Also, please specify the type, meaning this line gets `@param {object} options [...]`.

Reply:
Fixed in following commit